src/clang_parser.cpp

```diff
@@ -411,13 +411,6 @@ bool ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
             struct_name = ptypestr;
           remove_struct_union_prefix(struct_name);
 
-          // TODO(mmarchini): re-enable this check once we figure out how to
```

Comment:
IIRC this was commented out b/c clang didn't support flexible array members yet. Maybe leaving it in would still be useful to the next person coming along.

Reply:
the thing is that with the original fix in place, we fail to check incomplete arrays,
which returns 0 size and tools like runqlat.bt failed with:

`Can't get size of 'struct bucket_table::buckets', please provide proper definiton.`

That's why the quick fix at that point was to comment it out, with the note
it fails for incomplete arrays. The fix at that time was to add special check
for that, like:

```
-            if (clang_Type_getSizeOf(type) < 0) {
+            if (type.kind != CXType_IncompleteArray && clang_Type_getSizeOf(type) < 0) {

```

but meanwhile the original issue was solved with:
  364849d70279 clang_parser: return with error code if clang_parser finds an error

which handles incomplete arrays properly, so all this new commented code can go away